docs/cookbook/tracks/src/tracksplugin_coordinates.js

```diff
@@ -0,0 +1,106 @@
+goog.provide('plugin.cookbook_tracks.TracksPlugin');
+
+goog.require('ol.Feature');
+goog.require('os.plugin.AbstractPlugin');
+goog.require('os.plugin.PluginManager');
+goog.require('os.time.TimeInstant');
+goog.require('plugin.track');
+
+var transformToMap;
+
+/**
+ * Provides a plugin cookbook example for track creation and update.
+ *
+ * @extends {os.plugin.AbstractPlugin}
+ * @constructor
+ */
+plugin.cookbook_tracks.TracksPlugin = function() {
+  plugin.cookbook_tracks.TracksPlugin.base(this, 'constructor');
+  this.id = plugin.cookbook_tracks.ID;
+  this.errorMessage = null;
+};
+goog.inherits(plugin.cookbook_tracks.TracksPlugin, os.plugin.AbstractPlugin);
+
+plugin.cookbook_tracks.TracksPlugin.prototype.lat = -35.0;
+plugin.cookbook_tracks.TracksPlugin.prototype.lon = 135.0;
+plugin.cookbook_tracks.TracksPlugin.prototype.latDelta = 0.1;
+plugin.cookbook_tracks.TracksPlugin.prototype.lonDelta = 0.1;
+
+/**
+ * @type {string}
+ * @const
+ */
+plugin.cookbook_tracks.ID = 'cookbook_tracks';
+
+/**
+ * @inheritDoc
+ */
+plugin.cookbook_tracks.TracksPlugin.prototype.init = function() {
+  transformToMap = ol.proj.getTransform(os.proj.EPSG4326, os.map.PROJECTION);
+  var placesManager = plugin.places.PlacesManager.getInstance();
+  placesManager.listenOnce(os.config.EventType.LOADED, this.onPlacesLoaded, false, this);
```

Comment:
You may want the example to check `placesManager.isLoaded()` first. If it's already loaded `this.onPlacesLoaded` can be called immediately, otherwise the listener should be registered. While this shouldn't be necessary during plugin init (the places manager shouldn't be loaded yet), I think it's beneficial to show that pattern in the cookbook so readers are aware.

Reply:
Added to both source examples. Updated line numbers to match.